.travis.yml

```diff
@@ -52,11 +52,17 @@ deploy:
 after_deploy: bash .travis/release.sh
 
 notifications:
+  webhooks:
+    urls:
+      - https://webhooks.gitter.im/e/0e6dd310db54b96c2653
```

Comment:
Does this mean we'll get notifications on gitter during the build?

Reply:
You should see this on the right on gitter (where currently activity on github is shown). I've added it to both gitter channels (pmd/pmd and pmd/maintainers)